src/logger/config.ts

```diff
@@ -0,0 +1,62 @@
+import appRootDir from 'app-root-path'
+import { Configuration } from 'log4js'
+import { LoggerChannel } from './loggerChannel'
+import { LoggerConfig } from '@jolocom/sdk'
+import { OutputType } from './outputType'
+import { Level } from './level'
+
+const appendersConfigMap = (filePath: string) => ({
+  file: {
+    [LoggerChannel.SDK]: {
+      type: 'file',
+      filename: filePath + '/sdk.log',
+    },
+    [LoggerChannel.AGENT]: {
+      type: 'file',
+      filename: filePath + '/sdk_agent.log',
+    },
+    [LoggerChannel.INTERACTION]: {
+      type: 'file',
+      filename: filePath + '/sdk_interaction.log',
+    },
+  },
```

Comment:
shall we consider a case when a client wants to define its own log channel besides having sdk, agent and interaction channels. Can be passed via config

Reply:
This will not gives any benefits since within SDK must be defined explicitly places where those channels should be used (what to log with new channels).
In this case SDK defines places and channels which could be configured from the client side and capability to redefine implementation for the [Logger](https://github.com/jolocom/jolocom-sdk/blob/feat/logging/src/logger/logger.ts) it self by injecting them as a plugin (but of course client can inject (as a plugin) new, configured Logger instance and use it in other 'plugin' for example). So in this case we don't need to pass configuration thru one point and use it in plugged object from another point, it feels like smearing related logic within whole app.

Reply:
right, didn't think about possibility to inject custom logger as separate service/plugging. +1 for separation of concerns